content/rancher-prime/aws/contents.lr

```diff
@@ -0,0 +1,323 @@
+cattle-provisioning-capi-system     capi-controller-manager       1/1     1            1           8h
+cattle-system                       rancher                       3/3     3            3           8h
+cattle-system                       rancher-webhook               1/1     1            1           8h
+cert-manager                        cert-manager                  1/1     1            1           8h
+cert-manager                        cert-manager-cainjector       1/1     1            1           8h
+cert-manager                        cert-manager-webhook          1/1     1            1           8h
+default                             external-dns                  1/1     1            1           9h
+ingress-nginx                       ingress-nginx-controller      1/1     1            1           9h
+kube-system                         coredns                       2/2     2            2           20h
+kube-system                         coredns-autoscaler            1/1     1            1           20h
+kube-system                         extension-agent               1/1     1            1           8h
+kube-system                         extension-operator            1/1     1            1           8h
+kube-system                         konnectivity-agent            2/2     2            2           20h
+kube-system                         metrics-server                2/2     2            2           20h
+```
+
+
+Jobs and Pods:
+
+Check that pods or jobs have status Running/Completed
+
+
+```
+kubectl get pods --all-namespaces
+```
+
+
+if a pod is not in Running state, you can dig into the root cause by running:
+
+Describe pod
+
+
+```
+kubectl describe pod <pod name> -n <namespaces>
+```
+
+Pod container logs
+
+
+```
+kubectl logs <pod name> -n <namespaces>
+```
+
+Describe job
+
+
+```
+kubectl describe job <job name> -n <namespaces>
+```
+
+Logs from the containers of pods of the job
+
+```
+kubectl logs -l job-name=<job name> -n <namespaces>
+```
+
+###Recovery from Failed Pods
+
+If any of the pods is not Running : 
+
+Check the rancher-cloud Pod
+
+```
+kubectl get pods --all-namespaces | grep rancher-cloud
+```
+
+If the rancher-cloud pod is in Error state, wait for the pod to be deleted which takes approx 10 mins after it goes in Error State.
+
+Fix the problem and execute 
+
+```
+helm upgrade -n cattle-rancher-csp-deployer-system rancher-cloud --create-namespace \
+oci://709825985650.dkr.ecr.us-east-1.amazonaws.com/suse/rancher-payg/rancher-cloud-helm/rancher-cloud --install \
+  --version {{chart_version}} \
+  --set rancherHostname=$HOST_NAME \
+  --set rancherServerURL=https://$HOST_NAME \
+  --set rancherReplicas=$REPLICAS \
+  --set rancherBootstrapPassword=$BOOTSTRAP_PASSWORD \
+  --set global.aws.accountNumber=$AWS_ACCOUNT_ID \
+  --set global.aws.roleName=$ROLE_NAME
+```
+
+###ReRunning the installation because of customer Error
```

Comment:
I'd probably add a dash to make it Re-running rather than ReRunning... could go either way